src/bokeh/models/formatters.py

```diff
@@ -632,6 +641,142 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
+    | %micro   | Overall completed microseconds.                                 |
+    +----------+-----------------------------------------------------------------+
+    | %Milli   | Milliseconds since last second as a decimal number,             |
+    |          | zero-padded on the left (range 000 to 999).                     |
+    +----------+-----------------------------------------------------------------+
+    | %milli   | Overall completed milliseconds.                                 |
+    +----------+-----------------------------------------------------------------+
+    | %Second  | Seconds since last minute as a decimal number, zero-padded on   |
+    |          | the left (range 00 to 59).                                      |
+    +----------+-----------------------------------------------------------------+
+    | %second  | Overall completed seconds.                                      |
+    +----------+-----------------------------------------------------------------+
+    | %Minute  | Minutes since last hour as a decimal number, zero-padded on     |
+    |          | the left (range 00 to 59).                                      |
+    +----------+-----------------------------------------------------------------+
+    | %minute  | Overall completed minutes.                                      |
+    +----------+-----------------------------------------------------------------+
+    | %Hour    | Hours since last day as a decimal number, zero-padded on the    |
+    |          | left (range 00 to 23).                                          |
+    +----------+-----------------------------------------------------------------+
+    | %hour    | Overall completed hours.                                        |
+    +----------+-----------------------------------------------------------------+
+    | %day     | Overall completed days.                                         |
+    +----------+-----------------------------------------------------------------+
+
+    '''
+
+    # explicit __init__ to support Init signatures
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+
+    nanoseconds = String(help=_TIMEDELTA_TICK_FORMATTER_HELP("``nanoseconds``"),
+                          default="%Nanons")
+
+    microseconds = String(help=_TIMEDELTA_TICK_FORMATTER_HELP("``microseconds``"),
+                          default="%Microus")
+
+    milliseconds = String(help=_TIMEDELTA_TICK_FORMATTER_HELP("``milliseconds``"),
+                          default="%Millims")
+
+    seconds = String(help=_TIMEDELTA_TICK_FORMATTER_HELP("``seconds``"),
+                     default="%Hour:%Minute:%Second")
+
+    minsec = String(help=_TIMEDELTA_TICK_FORMATTER_HELP("``minsec`` (for combined minutes and seconds)"),
+                    default="%Hour:%Minute:%Second")
+
+    minutes = String(help=_TIMEDELTA_TICK_FORMATTER_HELP("``minutes``"),
+                     default="%Hour:%Minute")
```

Comment:
Not sure what change to make here, but I do think it will not be clear that e.g. `03:01` is not three minutes and one second rather than three hours and one minute.